mikelpardo/mobile-sales
# Get mobile detail from endpoint

**Description**

Get mobile detail from given endpoint

**How to test**

1. Go to: http://localhost:3000/
2. Click in one mobile item
3. Check if you have been redirect to mobile detail view and display all information correctly
